src/containers/shared/test/SocketContext.test.ts

```diff
@@ -19,18 +19,47 @@ describe('getSocket', () => {
 
   describe('server defined entrypoint', () => {
     beforeEach(() => {
+      delete process.env.VITE_RIPPLED_SECONDARY
```

Comment:
Why is this necessary? (Maybe worth a comment?)

Reply:
Does this have side effects if run on production? (i.e. is the process.env value permanently removed?)

Reply:
I will remove this.  This was originally there because I was going to have two environment variables.  Since all entry points are tried at once it made no sense for them to be called secondary.

Reply:
First off, this is a test, so it's not run in production. Second off, there's a `beforeEach` in the outside `describe` that saves and replaces `process.env`.